### PR TITLE
feat(query): Added Always Pull Images Admission Control Plugin Not Set for Kubernetes

### PR DESCRIPTION
**Pull request by cxMiguelSilva**
**Proposed Changes**
- Added Allways Pull Images Admission Control Plugin Not Set for Kubernetes

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/k8s/allways_pull_images_admission_control_plugin_not_set/metadata.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "a77f4d07-c6e0-4a48-8b35-0eeb51576f4f",
+  "queryName": "Allways Pull Images Admission Control Plugin Not Set",
```
Comment (rafaela-soares) [inline assets/queries/k8s/allways_pull_images_admission_control_plugin_not_set/metadata.json:3]:
```suggestion
  "queryName": "Always Pull Images Admission Control Plugin Not Set",
```

Please, also update the query folder name

File: assets/queries/k8s/allways_pull_images_admission_control_plugin_not_set/metadata.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  "severity": "MEDIUM",
+  "category": "Build Process",
+  "descriptionText": "The --enable-admission-plugins flag should have 'AllwaysPullImages' plugin and the plugin should be correctly configured in AdmissionControl Config file",
```
Comment (rafaela-soares) [inline assets/queries/k8s/allways_pull_images_admission_control_plugin_not_set/metadata.json:6]:
```suggestion
   "descriptionText": "When using kube-apiserver command, the '--enable-admission-plugins' flag should have 'AllwaysPullImages' plugin and the plugin should be correctly configured in AdmissionControl Config file",
```

File: assets/queries/k8s/allways_pull_images_admission_control_plugin_not_set/query.rego [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	args := container.args
+	startswith(args[a], flag)
+	values := split(args[a], "=")[1]
+	hasValue(values, value)
+}
+
+hasValue(values, value) {
+	splittedValues := split(values, ",")
+	splittedValues[_] == value
+}
```
Comment (rafaela-soares) [inline assets/queries/k8s/allways_pull_images_admission_control_plugin_not_set/query.rego:40]: As already suggested by @JoaoDanielRufino, you can add this code to the k8s library.

If you add it, please update `kics\assets\queries\k8s\authorization_mode_set_to_always_allow`

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Review by joaorufi — APPROVED** [review]

**Review by joaorufi — APPROVED** [review]

**Pull request merged**